app/main.py

```diff
@@ -1,3 +1,44 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.shop import Shop
+from app.customer import Customer
+
+
+def shop_trip() -> None:
+    with open("D:\\projects1\\py-shop-trip\\app\\config.json", "r") \
+            as data_file:
+        data_file_json = json.load(data_file)
+        # fuel_price = data_file_json["FUEL_PRICE"]
+        customers_list = []
+        for customer in data_file_json["customers"]:
+            customers_list.append(Customer(customer))
+
+        shop_list = []
+        for shop in data_file_json["shops"]:
+            shop_list.append(Shop(shop))
+
+        for shopper in customers_list:
+            print(f"{shopper.name} has {shopper.money} dollars")
+            dict_shop = {}
+            for shop in shop_list:
+                cost_to_shop = shopper.full_cost(
+                    shop,
+                    data_file_json["FUEL_PRICE"]
```

Comment:
You pass an int argument but need to receive the FuelPrice object
The same in all places